### DISCUSSION TITLE
Seems the "wire:click="function"" requires two clicks on mobile

### DISCUSSION THREAD

**Discussion by alnutile** [livewire/livewire#3005] [7. Bugs (archived)]
### Description
Seems the "wire:click="getCodeName"" requires two clicks on mobile browsers.
This StackOverflow mentions it but just curious if it is a bug or just me :)

https://stackoverflow.com/questions/32299475/why-do-my-onclick-functions-take-two-clicks/32299506

### Exact steps to reproduce

I have a super simple example app, you can see the code [here](https://github.com/alnutile/codenames/blob/main/resources/views/livewire/show-codename.blade.php#L16)

Works fine on web, you can see it [here](https://codenames.alfrednutile.info)

### Stripped-down, copy-pastable code snippets

```
            <button wire:click="getCodeName" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect" wire:loading.attr="disabled" style="min-width: 167.875px;">
                <span>Generate Another</span>
            </button>
```

**Context**
- Livewire version: v2.4.4
- Laravel version: v8.42.1
- Alpine version: [e.g. 2.3.0]
- Browser:Safari/Edge Mobile shows two click issue.

**Comment by nuernbergerA** [comment]
Hey @alnutile.

I tried it with my Android and Chrome without any issues.

**Comment by alnutile** [comment]
@nuernbergerA  ok that is a "real" os how about iOS :) 
Anyways I attached three screen-recordings, Safari, Chrome and Edge in iOS.


https://user-images.githubusercontent.com/365385/120084160-0cf8df80-c09c-11eb-95c7-a61282bb296a.mov

https://user-images.githubusercontent.com/365385/120084161-0e2a0c80-c09c-11eb-97cd-4dfc067ad559.mov

https://user-images.githubusercontent.com/365385/120084158-0bc7b280-c09c-11eb-9b07-b5b058df6c9a.mov

**Comment by nuernbergerA** [comment]
I think apple uses the same engine on iOS for all browsers. So chrome on iOS is the same as Safari.

**Comment by nuernbergerA** [comment]
@alnutile  try to give your button a `type="button"` maybe it helps
